src/components/Game.jsx

```diff
@@ -1,26 +1,106 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Board from './Board';
 
 const Game = () => {
 
+    let [history, setHistory] = useState([
+        {
+            squares: Array(9).fill(null)
+        }
+    ]);
+    let [stepNumber, setStepNumber] = useState(0);
+    let [xIsNext, setXIsNext] = useState(true);
+    const newHistory = history;
+    const currentSquares = newHistory[stepNumber].squares;
```

Comment:
In this case, you can use `history` directly.
```suggestion

    const currentSquares = history[stepNumber].squares;
```